src/pages/Main/Main.js

```diff
@@ -1,7 +1,161 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Slider from 'react-slick';
+import styled from 'styled-components';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import Mari from '../../assets/images/IMG_1211.JPG';
+import Category from '../../components/Category/Category';
 
 const Main = () => {
-  return <div>Main</div>;
+  const [rankingData, setRankingData] = useState([]);
+  const [categoryData, setCategoryData] = useState([]);
+  const settings = {
+    dots: true,
+    infinite: false,
+    speed: 500,
+    slidesToShow: 3,
+    slidesToScroll: 3,
+    rows: 3,
+    arrow: true,
+    draggable: true,
+  };
+  useEffect(() => {
+    fetch('/data/rankingList.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setRankingData(data);
+      });
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    fetch('/data/rankingList.json')
      .then(res => res.json())
      .then(data => {
        setRankingData(data);
      });
  }, []);
```
Get 요청일 때는 method 안 쓰셔도 됩니다!